app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/gtmCommunicatorFactory.ts

```diff
@@ -5,6 +5,8 @@ const gtmCommunicatorFactory = (window: Window): Communicator => ({
   sendPreferences: (userPreferences: UserPreferences) => {
     // eslint-disable-next-line no-param-reassign
     window.dataLayer = window.dataLayer || [];
+    window.dataLayer.push({ 'tracking-consent-frontend': 'true' });
```

Comment:
I think this needs to go here: https://github.com/hmrc/tracking-consent-frontend/blob/72e89d93887f0d4a145218828db01e5ceb2e5a6f/app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/enableGtm.ts#L13 so it will happen regardless of whether the user has clicked 'Accept all'